app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -232,11 +265,47 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
               {selectedLeague?.leagueName as string}
             </LinkCustom>
           </nav>
-          <section className="w-full pt-8" data-testid="weekly-picks">
+          <section
+            className="flex flex-col items-center w-full pt-8"
+            data-testid="weekly-picks"
+          >
             <h1 className="pb-8 text-center text-[2rem] font-bold text-foreground">
               Week {week} pick
             </h1>
 
+            {pickHistory.length > 0 && (
+              <section
+                className="flex flex-wrap w-[90%] gap-4 overflow-x-scroll justify-center pb-10 items-center"
+                data-testid="user-pick-history"
+              >
+                {pickHistory?.map((logoURL, index) => (
+                  <div
+                    key={`${logoURL}-${index + 1}`}
```

Comment:
From what I know about React, we shouldn't be using index as a key in React in a map function: [article about why it's bad to use index in a react map function](https://blog.anja-stricker.de/why-using-index-as-key-in-reactjs-map-function-can-cause-problems-and-how-to-fix-them]). Is there an alternative unique identifier we can use besides index?

Reply:
using index ALONE can cause issues. but it does not cause an issue when combined with something else like in this instance logoURL.